scripts/s2n_struct_mem.py

```diff
@@ -0,0 +1,94 @@
+#! /usr/bin/env python
```

Comment:
can you add the license header: https://github.com/aws/s2n-tls/blob/main/api/s2n.h#L1-L14